toot-relay.go

```diff
@@ -26,9 +26,10 @@ import (
 )
 
 type Message struct {
-	isProduction bool
-	notification *apns2.Notification
-	requestLog   *log.Entry // For logging with datadog context
+	isProduction   bool
+	notification   *apns2.Notification
+	unsubscribeUrl string
+	requestLog     *log.Entry // For logging with datadog context
```

Comment:
A little gross to have support for specific commercial non-OSS products wired into the code? Would this be log.Entry usable with a DataDog competitor?